frontend/src/components/creator-module/creator-answer-types/multiple-choice.scss

```diff
@@ -91,8 +98,21 @@
       display: flex;
       align-items: center;
       font-size: 1.25rem;
-      color: #ffc904;
-      text-shadow: 1px 1px black;
+      color: black;
+
+      &:hover {
+        background: whitesmoke;
+      }
+    }
+
+    .add-answer-icon {
+      font-size: 2rem;
+      font-weight: bold;
+      color: #ffc904
+    }
+
+    .add-answer-text {
```

Comment:
It seems like this part can be removed, since the only property in it is commented out.